backend/docs/manual/1-record-manager/1.9-vernietigingsrapport-configureren.rst

```diff
@@ -0,0 +1,69 @@
+.. _manual_1-record-manager_1.9-vernietigingsrapport-configureren:
+
+=================================
+Vernietigingsrapport configureren
+=================================
+
+.. figure:: ../_assets/pages/vernietigingsrapport-configureren.png
+   :align: center
+   :alt: Vernietigingsrapport configureren schermafbeelding
+
+Deze handleiding beschrijft de stappen die een **record manager** kan volgen om een vernietigingsrapport te configureren.
+
+Voorwaarden
+------------
+- De record manager moet beschikken over een actieve gebruikersaccount met de juiste toegangsrechten.
+- De volgende keuzes moeten beschikbaar zijn in het systeem:
+  - Informatieobjecttype
+  - Resultaattype
+  - Selectielijstklasse
+  - Statustype
+  - Zaaktype
```

Comment:
Maybe more meaningful to say that the services to talk to openzaak and the selectielijst API need to be configured? I had added (minimal) info on how to do this here: https://open-archiefbeheer.readthedocs.io/en/latest/configure/services.html